cartography/data/jobs/analysis/azure_sql_asset_exposure.json

```diff
@@ -5,13 +5,30 @@
 			"iterative": true,
 			"iterationsize": 100
 		},
+		{
+			"query": "MATCH ((n)<-[:RESOURCE]-(:AzureSQLServer)<-[:RESOURCE]-(:AzureSubscription{id: $AZURE_SUBSCRIPTION_ID})<-[:RESOURCE]-(:AzureTenant{id: $AZURE_TENANT_ID})<-[:OWNER]-(:CloudanixWorkspace{id: $WORKSPACE_ID})) WHERE EXISTS(n.exposed_internet) AND n:AzureSQLDatabase OR n:AzureRestorableDroppedDatabase OR n:AzureRecoverableDatabase WITH n limit $LIMIT_SIZE REMOVE n.exposed_internet_type RETURN COUNT(*) as TotalCompleted",
+			"iterative": true,
+			"iterationsize": 100
+		},
 		{
 			"query": "MATCH (sql:AzureSQLServer)<-[:RESOURCE]-(:AzureSubscription{id: $AZURE_SUBSCRIPTION_ID})<-[:RESOURCE]-(:AzureTenant{id: $AZURE_TENANT_ID})<-[:OWNER]-(:CloudanixWorkspace{id: $WORKSPACE_ID}) WHERE NOT (sql)-[:HAS]->(:AzureServerPrivateEndpointConnection) SET sql.exposed_internet = true, (CASE WHEN NOT 'no_private_endpoints' IN coalesce(sql.exposed_internet_type, []) THEN sql END).exposed_internet_type = coalesce(sql.exposed_internet_type, []) + 'no_private_endpoints';",
 			"iterative": false
 		},
 		{
 			"query": "MATCH (:AzureFirewallRule{start_ip_address: '0.0.0.0'})<-[:HAS]-(sql:AzureSQLServer)<-[:RESOURCE]-(:AzureSubscription{id: $AZURE_SUBSCRIPTION_ID})<-[:RESOURCE]-(:AzureTenant{id: $AZURE_TENANT_ID})<-[:OWNER]-(:CloudanixWorkspace{id: $WORKSPACE_ID}) SET sql.exposed_internet = true, (CASE WHEN NOT 'unrestricted_firewall' IN coalesce(sql.exposed_internet_type, []) THEN sql END).exposed_internet_type = coalesce(sql.exposed_internet_type, []) + 'unrestricted_firewall';",
 			"iterative": false
+		},
+		{
+			"query": "MATCH (db:AzureSQLDatabase)<-[:RESOURCE]-(sql:AzureSQLServer{exposed_internet: true})<-[:RESOURCE]-(:AzureSubscription{id: $AZURE_SUBSCRIPTION_ID})<-[:RESOURCE]-(:AzureTenant{id: $AZURE_TENANT_ID})<-[:OWNER]-(:CloudanixWorkspace{id: $WORKSPACE_ID}) SET db.exposed_internet = true, db.exposed_internet_type = sql.exposed_internet_type;",
+			"iterative": false
+		},
+		{
+			"query": "MATCH (db:AzureRestorableDroppedDatabase)<-[:RESOURCE]-(sql:AzureSQLServer{exposed_internet: true})<-[:RESOURCE]-(:AzureSubscription{id: $AZURE_SUBSCRIPTION_ID})<-[:RESOURCE]-(:AzureTenant{id: $AZURE_TENANT_ID})<-[:OWNER]-(:CloudanixWorkspace{id: $WORKSPACE_ID}) SET db.exposed_internet = true, db.exposed_internet_type = sql.exposed_internet_type;",
```

Comment:
we do not need to define public facing for `AzureRestorableDroppedDatabase` and `AzureRecoverableDatabase`. Please remove this code.